src/sentry/options/store.py

```diff
@@ -92,6 +89,9 @@ def get_cache(self, key, silent=False):
         if value is not None:
             return value
 
+        if self.cache is None:
+            return None
```

Comment:
Nah, this is fine, because it just acts like a cache miss. `self.cache.get()` would return `None` as well on a miss.